File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.py
import sys

input = sys.stdin.readline

N, M = map(int, input().split())
pw = {}

for _ in range(N):
    site, password = input().split()
    pw[site] = password

out = []
for _ in range(M):
    query = input().strip()
    out.append(pw[query])

print("\n".join(out))

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        StringTokenizer st = new StringTokenizer(br.readLine());
        int N = Integer.parseInt(st.nextToken());
        int M = Integer.parseInt(st.nextToken());

        Map<String, String> pwMap = new HashMap<>();

        for (int i = 0; i < N; i++) {
            st = new StringTokenizer(br.readLine());
            String site = st.nextToken();
            String password = st.nextToken();
            pwMap.put(site, password);
        }

        for (int i = 0; i < M; i++) {
            String query = br.readLine().trim();
            sb.append(pwMap.get(query)).append("\n");
        }

        System.out.print(sb);
    }
}